Adds A Sewing Recipe for Standard Hood

Sulaboy opened the pull request:
I hope this isn't considered balance.

Recipe is the same as cloth mask since they do the same thing, just in different slots.
1 Cloth
1 Fiber
0 Crafting Difficulty

Also please let me know if I screwed up making this PR, Github confounds me.

<!-- Write **BELOW** The Headers and **ABOVE** The comments else it may not be viewable. -->
<!-- Note: PRs including balance changes authored by anyone other than maintainers and official devs will not be considered. -->

## About The Pull Request
Adds a recipe to be able to sew the default cloth hood.
<!-- Describe The Pull Request. Please be sure every change is documented or this can delay review or prevent the PR from being merged! -->

## Why It's Good For The Game
This lets players sew cloth hoods, for whatever reason they might want to.
<!-- Please add a short description of why you think these changes would benefit the game. If you can't justify it in words, it might not be worth adding. -->

## Pre-Merge Checklist
<!-- Don't bother filling these in while creating your Pull Request, just click the checkboxes after the Pull Request is opened and you are redirected to the page. -->
- [x] You tested this on a local server.
- [x] This code did not runtime during testing.
- [x] You documented all of your changes.
<!-- Neither the compiler nor workflow checks are perfect at detecting runtimes and errors. It is important to test your code/feature/fix locally. -->

Sulaboy commented:
> ```
> /datum/repeatable_crafting_recipe/leather/hood
> 	name = "leather hood"
> 	output = /obj/item/clothing/head/roguehood/brown
> 	craftdiff = 2
> ```

I wasn't sure what you meant by this, but I realized I didn't set it to the cloth color subtype, so that's fixed.

Sadboysuss commented:
> I wasn't sure what you meant by this, but I realized I didn't set it to the cloth color subtype, so that's fixed.

there is already a crafting recipe for this and it's called leather hood.

Sulaboy commented:
> > I wasn't sure what you meant by this, but I realized I didn't set it to the cloth color subtype, so that's fixed.
> 
> there is already a crafting recipe for this and it's called leather hood.

The whole reason I made this pr was because I was under the impression that the standard hoods that spawned round start were cloth. 

I just tested it and turns out they're not. What is even sillier is I found out if you have any skincraft skill you can dupe leather with the hoods by snipping them (gives a whole raw hide instead of cured leather) and sewing them over and over again.

Hoods don't give any armor that I could see, and they just hide identity. Would you be open to me removing the leather variant, and making it only able to be crafted with cloth? The hoods look like they're made out of cloth, and there's nothing to justify it taking cured leather (which tailors run out of very often). I'd also fix the recycle duplication glitch.

Sulaboy commented:
I bumped up the craftdiff to 1 so it's in line with a cloth coif, and removed the leather hood recipe. I also changed the salvage return to be 1 cloth so no duping.

Sadboysuss commented on code/modules/crafting/quality_of_crafting/sewing.dm:
```diff
@@ -53,6 +53,7 @@
 	)
 	output = /obj/item/clothing/cloak/apron
 
+
```
```suggestion
```
